src/Conferences/ConferencesHarvester.php

```diff
@@ -86,6 +85,11 @@ public function harvest(): void
 
         $this->logger->notice($newConferencesCount.' newly added conference(s)');
         $this->logger->notice($updatedConferencesCount.' updated conference(s)');
+
+        return [
+            'newConferences' => $newConferencesCount,
+            'updatedConferences' => $updatedConferencesCount,
```

Comment:
```suggestion
            'newConferencesCount' => $newConferencesCount,
            'updatedConferencesCount' => $updatedConferencesCount,
```

Even for testing, it's important to use a good name :)